invitations/api/endpoint_test.go

```diff
@@ -543,3 +543,74 @@ func TestAcceptInvitation(t *testing.T) {
 		repoCall.Unset()
 	}
 }
+
+func TestRejectInvitation(t *testing.T) {
+	is, svc := newIvitationsServer()
+
+	cases := []struct {
+		desc        string
+		token       string
+		data        string
+		contentType string
+		status      int
+		svcErr      error
+	}{
+		{
+			desc:        "valid request",
+			token:       validToken,
+			data:        fmt.Sprintf(`{"domain_id": "%s"}`, validID),
+			status:      http.StatusNoContent,
+			contentType: validContenType,
+			svcErr:      nil,
+		},
+		{
+			desc:        "invalid token",
+			token:       "",
+			data:        fmt.Sprintf(`{"domain_id": "%s"}`, validID),
+			status:      http.StatusUnauthorized,
+			contentType: validContenType,
+			svcErr:      nil,
+		},
+		{
+			desc:        "with service error",
```

Comment:
Either use `with` everywhere or remove it from here. While it is redundant, I prefer having it, but I leave it up to you, just stay consistent.